Userland/Libraries/LibGPU/Image.cpp

```diff
@@ -0,0 +1,34 @@
+/*
+ * Copyright (c) 2022, Stephan Unverwerth <s.unverwerth@serenityos.org>
+ *
+ * SPDX-License-Identifier: BSD-2-Clause
+ */
+
+#include <AK/Math.h>
+#include <LibGPU/Image.h>
+
+namespace GPU {
+
+Image::Image(void const* ownership_token, GPU::PixelFormat const& pixel_format, u32 width, u32 height, u32 depth, u32 max_levels)
+    : m_ownership_token { ownership_token }
+    , m_pixel_format { pixel_format }
+{
+    VERIFY(width > 0);
+    VERIFY(height > 0);
+    VERIFY(depth > 0);
+    VERIFY(max_levels > 0);
+
+    using AK::log2;
```

Comment:
Don't we usually place these at the top of the file?

Reply:
I put the using directly in the function since it's the only place where it is used. I can pull it into the file scope it you want.

Reply:
Apparently our coding style indicates we should not do this at all:
https://github.com/SerenityOS/serenity/blob/master/Documentation/CodingStyle.md#using-statements

> In C++ implementation files, do not use "using" declarations of any kind to import names in the standard template library. Directly qualify the names at the point they're used instead.

Reply:
Ok I will change this then.